src/pineko/evolve.py

```diff
@@ -142,6 +143,13 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
     operators_card["eko_version"] = metadata.version("eko")
     pineko_version = metadata.version("pineko")
 
+    # switch on polarization, only in DIS
+    if "runcard" in pineappl_grid.key_values():
+        grid_runcard = pineappl_grid.key_values()["runcard"]
+        # here a grid contains a single obrevable
+        observable = list(json.loads(grid_runcard)["observables"])[0]
+        operators_card["configs"]["polarized"] = "g" in observable.split("_")[0]
```

Comment:
The following is very minor but might be slightly restrictive.

```suggestion
        operators_card["configs"]["polarized"] = observable.startswith("g")
```